pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/Constants.java

```diff
@@ -59,6 +61,18 @@ public static TableType validateTableType(String tableTypeStr) {
     }
   }
 
+  public static ComponentType validateComponentType(String componentTypeStr) {
+    if (componentTypeStr == null || componentTypeStr.isEmpty()) {
+      return null;
+    }
```

Comment:
you could us `StringUtils.isNotNull` here

Reply:
Thx， I have accepted your advice. @apucher